libs/langchain/langchain/callbacks/streaming_iter.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+import threading
+from queue import Queue
+from typing import Any, Dict, Iterator, List, Union
+
+from langchain.callbacks.base import BaseCallbackHandler
+from langchain.schema.output import LLMResult
+
+
+class IteratorCallbackHandler(BaseCallbackHandler):
+    """Callback handler that returns an async iterator."""
+
+    queue: Queue[str]
+
+    done: threading.Event
```

Comment:
Are these meant to be instance variables rather than class variables?

Reply:
I'm deleting this in a bit hopefully